docs/cmdline-opts/limit-rate.md

```diff
@@ -15,6 +15,7 @@ Example:
   - --limit-rate 100K $URL
   - --limit-rate 1000 $URL
   - --limit-rate 10M $URL
+  - --limit-rate 200K --max-time 60 -o $URL :Aborts if download takes longer than 60 seconds
```

Comment:
I don't think we should put comments in that line. It was not intended and I imagine it will make the output look wrong in some places.